package.json

```diff
@@ -18,8 +18,8 @@
     "dev": "vite --host",
     "build": "tsc && vite build",
     "preview": "vite preview",
-    "test": "jest",
-    "test:watch": "jest --watch --no-cache",
+    "test": "TZ=UTC jest",
+    "test:watch": "TZ=UTC jest --watch --no-cache",
```

Comment:
I think this configuration may cause some problems, so I would recommand to set the time zone directly in the test to improve its determinism.

Reply:
Definitely agree to limit timezone config to the test.

It has been defined [directly in the test](https://github.com/okp4/dataverse-portal/compare/b4eeaa4f10dad3c6d2718743dd8fb5c1891c7ee4..dee74bba8e639ac7d0e18bbe212034156f4dd7ef#diff-e605af1e3f46144bd488dd4893377c8f45ca2afd8f43ddd14011db3c288ea8e8R81) according to your feedback.

Thanks!